railties/lib/rails/generators/rails/app/templates/Gemfile.tt

```diff
@@ -10,9 +10,11 @@ ruby <%= "\"#{RUBY_VERSION}\"" -%>
 
 # Use Kredis to get higher-level data types in Redis [https://github.com/rails/kredis]
 # gem "kredis"
+<% end -%>
+<% unless options.skip_bcrypt? -%>
 
 # Use Active Model has_secure_password [https://guides.rubyonrails.org/active_model_basics.html#securepassword]
-# gem "bcrypt", "~> 3.1.7"
+<%= comment_if :skip_bcrypt_is_nil %>gem "bcrypt", "~> 3.1.7"
```

Comment:
The test failure appears to be real:

```
NoMethodError: undefined method `comment_if' for #<#<Class:0x00007f0e515e9d70>:0x00007f0e51330070 @output_buffer="source \"https://rubygems.org\"\ngit_source(:github) { |repo| \"https://github.com/\#{repo}.git\" }\n\nruby \"3.1.2\"\n\n# Use sqlite3 as the database for Active Record\ngem 'sqlite3'\n\n\n# Use Kredis to get higher-level data types in Redis [https://github.com/rails/kredis]\n# gem \"kredis\"\n\n# Use Active Model has_secure_password [https://guides.rubyonrails.org/active_model_basics.html#securepassword]\n">
```

Reply:
@skipkayhil Thanks for the feedback! I had put this in because I thought it would be helpful for novice users to be able to include `bcrypt` while still having a `minimal` build--a situation that occurs frequently in my context of using Rails to teach programming--and there seemed to be a precedent for treating individual gems on the CLI (e.g., `jbuilder`). But I see your point, and it's not too much trouble simply to add `gem "bcrypt"` to the built application. I'll go ahead and close the pr.